src/tensor_api.cpp

```diff
@@ -199,6 +199,33 @@ extern "C" miopenStatus_t miopenSetTensorDescriptor(miopenTensorDescriptor_t ten
     });
 }
 
+extern "C" miopenStatus_t miopenSetTensorDescriptorEx(miopenTensorDescriptor_t tensorDesc,
+                                                      miopenDataType_t dataType,
+                                                      int nbDims,
```

Comment:
`nbDims` cannot be negative. Wouldn't it be better to use `unsigned` instead?

Reply:
To stay compatible with `miopenSetTensorDescriptor` (no warnings at the user's side after migration to Ex)